sample/SampleHost/Functions.cs

```diff
@@ -61,6 +62,12 @@ public Functions(ISampleServiceA sampleServiceA, ISampleServiceB sampleServiceB)
             log.LogInformation($"Message complete (Id={messageId})");
         }
 
+        /// <summary>
+        /// If there is an error during function invocation, function execution will be retried up to 3 times waiting for "00:00:20" between each retry
+        /// </summary>
+        /// <param name="events"></param>
+        /// <param name="log"></param>
+        [FixedDelayRetryAttribute(3, "00:00:20")]
```

Comment:
nit: remove the unecessary "Attribute" from the definition. Also, can you change the retry interval to 3 seconds?